tests/test_model.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from sqlmodel import Session, SQLModel, create_engine
+
+from snipster.models import Snippet
+
+engine = create_engine("sqlite:///:memory", echo=True)
```

Comment:
I would encapsulate this in `setup_database` but it does not matter much here, because we're not going to import from this file